src/documents/tests/test_document_model.py

```diff
@@ -55,6 +56,24 @@ def test_file_name(self):
         )
         self.assertEqual(doc.get_public_filename(), "2020-12-25 test.pdf")
 
+    def test_file_name_with_timezone(self):
+
+        doc = Document(
+            mime_type="application/pdf",
+            title="test",
+            created=timezone.datetime(
+                2020,
+                12,
+                25,
+                0,
+                0,
+                0,
+                0,
+                pytz.timezone("Europe/Berlin"),
+            ),
+        )
+        self.assertEqual(doc.get_public_filename(), "2020-12-25 test.pdf")
```

Comment:
I always like to see more testing.  Do you think it would be beneficial to add some additional timezone/date combinations to check?

Reply:
@stumpylog The core issue described in #267 is that the date for the document always is received from an UTC point of view, which doesn't match users with different timezones, esp. the timezones "UTC+1, UTC+2, ..." because then you have that offset by one. This test covers this scenario. 

Is there another edgecase you see which would be worth testing?

Reply:
I'm alright with just the 1 case then

Reply:
I'm alright with just the 1 case then